### DISCUSSION TITLE
Is it possible to get translation messages from a CSV file?

### DISCUSSION THREAD

**Discussion by phillipmohr** [nuxt-modules/i18n#3200] [Q&A] [answered]
Hi, I was wondering if it is possible to read a CSV instead of a Json file.

Something like 
```js
import translations from './public/locales/translations.csv'
```

I was also having a look at the nuxt translation manager https://nuxt.com/modules/translation-manager but unfortunately the module does not allow external CSV files, which I need in my case.

**Comment by BobbieGoede** [comment, marked as answer]
This module doesn't support csv files and  at the moment we don't have plans to add support for it, sorry!

It would be best if translation-manager supported external csv files and I see you have already opened an issue there regarding external csv files, hopefully it gets implemented soon and otherwise I'm sure they would appreciate anyone opening a PR that adds the functionality💪
